RecoEgamma/Configuration/python/RecoEgamma_EventContent_cff.py

```diff
@@ -192,11 +71,67 @@
 #HI-specific products needed in pp scenario special configurations
 for e in [pA_2016, peripheralPbPb, pp_on_AA_2018, pp_on_XeXe_2017, ppRef_2017]:
-    for ec in [RecoEgammaAOD.outputCommands, RecoEgammaRECO.outputCommands, RecoEgammaFEVT.outputCommands]:
-        e.toModify( ec, func=lambda outputCommands: outputCommands.extend(['keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppGED_*_*',
-                                                                           'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerpp_*_*',
-                                                                           'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppIsland_*_*',
-                                                                           'keep recoPhotonCores_islandPhotonCore_*_*',
-                                                                           'keep recoPhotons_islandPhotons_*_*'
-                                                                           ])
-                    )
+    e.toModify( RecoEgammaAOD.outputCommands, 
+                func=lambda outputCommands: outputCommands.extend(['keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppGED_*_*',
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerpp_*_*',
+                                                                   'keep recoHIPhotonIsolationedmValueMap_photonIsolationHIProducerppIsland_*_*',
+                                                                   'keep recoPhotonCores_islandPhotonCore_*_*',
+                                                                   'keep recoPhotons_islandPhotons_*_*'])
+               )
+# RECO content
+RecoEgammaRECO = cms.PSet(
+    outputCommands = cms.untracked.vstring(
+        'keep *_gedPhotonCore_*_*',
+        'keep *_gedPhotons_*_*',
+        #'keep *_gedPhotonsTmp_*_*',        
+        'keep recoPhotons_mustachePhotons_*_*',
+        'keep recoPhotonCores_mustachePhotonCore_*_*',
+        #'drop *_gedPhotonsTmp_valMapPFEgammaCandToPhoton_*',
+        'keep recoTrackExtras_ckfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_ckfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_ckfInOutTracksFromConversions_*_*',
+        'keep recoConversions_uncleanedOnlyAllConversions_*_*',
+        'keep recoTrackExtras_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep recoTrackExtras_uncleanedOnlyCkfInOutTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfOutInTracksFromConversions_*_*', 
+        'keep TrackingRecHitsOwned_uncleanedOnlyCkfInOutTracksFromConversions_*_*',
+        'keep recoRecoEcalCandidates_hfRecoEcalCandidate_*_*')                                                                 
```

Comment:
In the AOD content, at lines 41-42 there is:
```PYTHON
        'keep *_hfRecoEcalCandidate_*_*',
        'keep recoRecoEcalCandidates_hfRecoEcalCandidate_*_*',
```
Therefore, in principle this additional ```keep``` statement for the RECO event content is not needed.

However, please check with EGAMMA contacts (@afiqaize @SohamBhattacharya) whether the   ```'keep *_hfRecoEcalCandidate_*_*',``` was really intended in the AOD content. If not (as I suspect) it should be added back to the FEVT event content here, then.